tools/kagami/src/genesis.rs

```diff
@@ -76,58 +79,66 @@ impl<T: Write> RunArgs<T> for Args {
 pub fn generate_default(
     builder: RawGenesisBlockBuilder<executor_state::SetPath>,
 ) -> color_eyre::Result<RawGenesisBlockFile> {
+    let genesis_account_id: AccountId = {
+        let multihash =
+            std::env::var("GENESIS_PUBLIC_KEY").wrap_err("GENESIS_PUBLIC_KEY should be set")?;
+        let domain = &*GENESIS_DOMAIN_ID;
+        format!("{multihash}@{domain}")
+            .parse()
+            .wrap_err("GENESIS_PUBLIC_KEY should be valid multihash")?
+    };
```

Comment:
To make it uniform with the rest of Kagami, please define it as a Clap argument of the `genesis` subcommand (`--genesis-public-key`/`GENESIS_PUBLIC_KEY`)